### PR TITLE
feat: move extension to open source

### PR DESCRIPTION
**Pull request by louisjoecodes**

### CODE REVIEW

File: .hacking/scripts/build_for_releases.sh [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+# Iterate over the architectures and operating systems
+archs=("amd64" "arm64")
+oses=("linux" "darwin")
+
+for arch in "${archs[@]}"; do
+  for os in "${oses[@]}"; do
+    # Set the output binary name
+    output_binary="$build_dir/${project_name}_${os}_${arch}_${version}"
+
```
Comment (benfdking) [inline .hacking/scripts/build_for_releases.sh:28]: This can be deleted

File: .github/workflows/rust-lint.yml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+jobs:
+  rust-lint:
+    name: Rust Lint
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@0ad4b8fadaa221de15dcec353f45205ec38ea70b # ratchet:actions/checkout@v3
+        with:
+          submodules: 'true'
+      - name: Install stable toolchain
+        uses: actions-rs/toolchain@16499b5e05bf2e26879000db0c1d13f7e13fa3af # ratchet:actions-rs/toolchain@v1
```
Comment (benfdking) [inline .github/workflows/rust-lint.yml:19]: This duplicates the job

File: .github/workflows/pr.yml [hunk 3]
```diff
@@ -20,6 +20,85 @@ jobs:
...
+      - name: Set up Homebrew
+        id: set-up-homebrew
+        uses: Homebrew/actions/setup-homebrew@35cdeaf60df48fb1fb7eac25fea4ebde1f72fc46 # ratchet:Homebrew/actions/setup-homebrew@master
+      - name: Install buf
+        run: brew install bufbuild/buf/buf
+      - run: make proto
+      - uses: Swatinem/rust-cache@23bce251a8cd2ffc3c1075eaa2367cf899916d84 # ratchet:Swatinem/rust-cache@v2
+      - uses: actions-rs/cargo@844f36862e911db73fe0815f00a4a2602c279505 # ratchet:actions-rs/cargo@v1
+        with:
+          command: check
```
Comment (benfdking) [inline .github/workflows/pr.yml:101]: This duplicates quite a lot of work that is already done.

File: .github/workflows/pr.yml [hunk 4]
```diff
@@ -116,10 +195,27 @@ jobs:
...
+      - name: Setup Node
+        uses: actions/setup-node@60edb5dd545a775178f52524783378180af0d1f8 # ratchet:actions/setup-node@v3
+      - uses: pnpm/action-setup@a3252b78c470c02df07e9d59298aecedc3ccdd6d # ratchet:pnpm/action-setup@v2
+        name: Install pnpm
+        id: pnpm-install
+        with:
+          version: 8
+          run_install: false
+      - name: Install dependencies
+        run: pnpm install --filter "@quary/root"
```
Comment (benfdking) [inline .github/workflows/pr.yml:220]: This only slows the job down because the make command uses npx

**Review by benfdking — CHANGES_REQUESTED** [review]
Few little changes

File: .github/workflows/pr.yml (hunk 4 above)
Comment (louisjoecodes) [reply .github/workflows/pr.yml:220]: We introduce `prettier-plugin-tailwindcss`, npx prettier depends on `prettier-plugin-tailwindcss` to work. The pnpm install step makes sure that this dependency is installed.

**Pull request merged**